apps/transport/priv/repo/migrations/20231019121309_data_conversion_add_columns.exs

```diff
@@ -0,0 +1,38 @@
+defmodule DB.Repo.Migrations.DataConversionAddColumns do
+  use Ecto.Migration
+
+  def up do
+    alter table("data_conversion") do
+      add :status, :string
+      add :converter, :string
+      add :converter_version, :string
+    end
+
+    drop unique_index("data_conversion", [:convert_from, :convert_to, :resource_history_uuid])
+
+    execute "update data_conversion set status = 'success'"
+    # https://github.com/etalab/transport-tools/blob/278f66679537981176f414de8027a86b1b0808ba/Dockerfile#L38
+    execute "update data_conversion set converter = 'hove/transit_model', converter_version = '0.55.0' where convert_to = 'NeTEx' and convert_from = 'GTFS'"
+    # https://github.com/etalab/transport-tools/blob/278f66679537981176f414de8027a86b1b0808ba/Dockerfile#L8
+    execute "update data_conversion set converter = 'rust-transit/gtfs-to-geojson', converter_version = '9ca9a25b895ba1b2fdf4d04e92895afec52d0608' where convert_to = 'GeoJSON' and convert_from = 'GTFS'"
```

Comment:
Cf ci-dessus, je me demande si utiliser le numéro de version ne serait pas plus sympa

https://github.com/rust-transit/gtfs-to-geojson/blob/9ca9a25b895ba1b2fdf4d04e92895afec52d0608/Cargo.toml#L3

Reply:
Traité https://github.com/etalab/transport-site/pull/3549#discussion_r1368816097